src/app/public/modules/inline-delete/inline-delete.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+.sky-inline-delete {
+  position: absolute !important;
```

Comment:
Would you mind putting in a comment for some context around the `!important` ?

Reply:
Done